install/kubernetes/cilium/templates/cilium-agent/dashboards-configmap.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.agent.dashboards.enabled }}
```

Comment:
Thanks for fixing the Helm values, this template needs to be updated too (`.Values.dashboards` instead of `.Values.agent.dashboards` everywhere)

Reply:
oops, fixed.